masu/config.py

```diff
@@ -37,6 +37,15 @@ class Config(object):
     DB_HOST = os.getenv('DATABASE_HOST', 'localhost')
     DB_PORT = os.getenv('DATABASE_PORT', 15432)
 
+    # Koku Connectivity
+    KOKU_HOST = os.getenv('KOKU_HOST_ADDRESS', 'localhost')
+    KOKU_PORT = os.getenv('KOKU_PORT', 8000)
+    KOKU_ADMIN_USER = os.getenv('SERVICE_ADMIN_USER', 'admin')
+    KOKU_ADMIN_PASS = os.getenv('SERVICE_ADMIN_PASSWORD', 'pass')
+
+    # Set method for retreiving CUR accounts. 'db' or 'network'
+    ACCOUNT_ACCESS_TYPE = os.getenv('ACCOUNT_ACCESS_TYPE', 'db')
```

Comment:
I don't know.  I sort of like being able to easily set default values to the individual components in their own environment variable rather than having one big string to establish the default behavior.

Maybe we should save this for if we end up switching to the microservice architecture and the individual environment variables prove to be too onerous.